libs/src/services/creatorNode/index.js

```diff
@@ -45,6 +45,23 @@ class CreatorNode {
     return null
   }
 
+  /**
+   * Checks if a cid is available from provided creator node endpoint
+   * @param {string} endpoint creator node endpoint
+   * @param {number} cid
+   */
+  static async checkIfCidAvailable (endpoint, cid) {
+    try {
+      return (await axios({ url: `ipfs/${cid}`, baseURL: endpoint, method: 'head' })).status === 200
```

Comment:
is it possible to use this function instead of adding another axios get function? https://github.com/AudiusProject/audius-protocol/blob/d371c76b202babc4281cca040469b177c303a649/libs/src/utils/network.js

Reply:
I think this will make a full request instead of a head request; that is why I think having a new get function might be a better idea.